integration/fuse/bin/alluxio-fuse

```diff
@@ -89,17 +89,22 @@ umount_fuse() {
     return 1
   else
     kill ${fuse_pid}
-    local maxWait=60
+    local maxWait=120
     local cnt=${maxWait}
     while kill -0 ${fuse_pid} > /dev/null 2>&1; do
       if [[ ${cnt} -gt 1 ]]; then
         # still not dead, wait
         cnt=$(expr ${cnt} - 1)
         sleep 1
-      else
+      elif [[ "${SOFT_KILL_OPT}" = true ]] ; then
         # waited long enough, giving up umount
-        echo "Failed to umount fuse process ["${fuse_pid}"] after waiting for "${maxWait}" seconds." >&2
+        echo "Soft kill option '-s' is specified. Failed to umount fuse process ["${fuse_pid}"] after waiting for "${maxWait}" seconds." >&2
```

Comment:
Failed to umount fuse process ["${fuse_pid}"] after waiting for "${maxWait}" seconds. Fuse device is busy and soft kill option `-s` is specified.